milabench/validation/usage.py

```diff
@@ -0,0 +1,84 @@
+from collections import defaultdict
+
+from .validation import ValidationLayer
+
+
+class _Layer(ValidationLayer):
+    """Checks that GPU utilisation is > 0.01 and that memory used is above 50%.
+
+    Notes
+    -----
+    This is a sanity check that only runs when enabled
+
+    """
+
+    def __init__(self, **kwargs) -> None:
+        self.warnings = defaultdict(lambda: defaultdict(float))
+
+        self.devices = set()
+        self.count = 0
+        self.mem_threshold = 0.50
+        self.load_threshold = 0.01
```

Comment:
We can leave it at that for now, but it'd be nice to make this configurable. I think the ideal place to do that would actually be in the benchmark's configuration, since we could adjust the expectations depending on the benchmark.

Also, `gpu_load/mem_threshold` would be a better naming convention. And we can track CPU/RAM as well in a next update.

Reply:
done, added gpu_ to the thresholds